packages/amplify-graphql-api-construct-tests/src/__tests__/migration/migration-validation.test.ts

```diff
@@ -60,24 +60,30 @@ describe('Migration table import validation', () => {
     deleteProjectDir(gen1ProjRoot);
   });
 
-  /*
-  AttributeDefintions - done
-  KeySchema
-  GlobalSecondaryIndexes - done
-  BillingModeSummary - done 
-  ProvisionedThroughput
-  StreamSpecification
-  SSEDescription
-  DeletionProtectionEnabled
-  */
-
   // Test cases for test.each need to be defined before beforeAll.
   // The tests cases need data defined in beforeAll.
   // Define the test name first then get the definition later (after beforeAll).
   // For each test case, there should be a matching key in the test definitions.
   // The test definitions include a schema, overrides, and the expected error messages.
-  const testCases = ['extraGSIOnGen2', 'billingMode'];
-  const getTestDefinition = (testCaseName: string): [TestDefinition, string, string[]] => {
+  const testCases = [
```

Comment:
Could you derive this with `Object.keys(testDefinitions)`?

Reply:
I'm going to refactor this further since I now realize that the test definition is the same for each test.